docs/language/overview.md

```diff
@@ -152,23 +151,23 @@ on values as they are produced.  For example, a long running query that
 produces incremental output will stream results as they are produced, i.e.,
 running `zq` to standard output will display results incrementally.
 
-The `search` and `where` operators "find" values in their input and drop
+The [`search`](operators/search.md) and [`where`](operators/where.md) operators "find" values in their input and drop
```

Comment:
@nwt: Sure, happy to accept all these wrapping suggestions. I'm sometimes unsure where people land on this topic. I've known some reviewers to get annoyed when formatting-only changes accompany meaningful changes in content and hence prefer some long lines rather than additional lines being flagged in the diff. I also wasn't sure where your preferences were on follow-on wraping changes, e.g., by accepting just this suggestion, now the next line becomes way shorter than 80 columns, even though it has a line break mid-sentence, so I wasn't sure if I'd then want to reformat the rest of the paragraph, but I think I see your preferences now. I'm fine with whatever, so I'll try to remember this for future passes in docs.